### PR TITLE
Hatskier/streamr broadcaster

### PR DESCRIPTION
**Pull request by hatskier**
[Streamr](https://streamr.network/) broadcaster implementation. Streamr is a decentralized pub sub network that allows to publish data for free. It will serve as a backup data source in the [redstone-flash-storage](https://github.com/redstone-finance/redstone-flash-storage)

The `streamr-client` module caused some problems during the code compilation with tsc, so I've added "skipLibCheck": true to the compiler options in the tsconfig.json. I had not found a better solution and [posted a question on the Streamr discord](https://discord.com/channels/801574432350928907/802079966350802944/912539312913010709). Will fix it if they reply :)

I've also upgraded the node version to 14 in Dockerfiles and made a major update of npm dependencies

I also created a test streamr product https://streamr.network/marketplace/products/3f5b8f7c2c0a4d9ab17724b2814457c48c8862917f8f4bacbd64193232973dca

### CODE REVIEW

File: src/broadcasters/streamr/StreamrBroadcaster.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+const logger = require("../../utils/logger")("StreamrBroadcaster") as Consola;
+
+const STREAMR_BROADCASTING_INTERVAL_MILLISECONDS = 3000;
+const PACKAGE_STREAM_NAME = "package";
+const PRICES_STREAM_NAME = "prices";
+
+export class StreamrBroadcaster implements Broadcaster {
+  private streamrProxy: StreamrProxy;
+  private timer: any;
```
Comment (ppedziwiatr) [inline src/broadcasters/streamr/StreamrBroadcaster.ts:14]: `any` -> `NodeJS.Timer`?

File: src/NodeRunner.ts [hunk 2]
```diff
@@ -219,9 +225,14 @@ export default class NodeRunner {
     logger.info("Broadcasting prices");
     const broadcastingTrackingId = trackStart("broadcasting");
     try {
-      await this.broadcaster.broadcast(signedPrices);
+      const promises = [];
+      promises.push(this.httpBroadcaster.broadcast(signedPrices));
+      if (this.nodeConfig.enableStreamrBroadcaster) {
+        promises.push(this.streamrBroadcaster.broadcast(signedPrices));
+      }
+      await Promise.all(promises);
```
Comment (ppedziwiatr) [inline src/NodeRunner.ts:233]: wonder if we shouldn't switch to `Promise.allSettled` - as whole `Promise.all` rejects as soon as one of the promises rejects - but maybe that's what we want here?

**Review by ppedziwiatr — APPROVED** [review]

File: src/broadcasters/streamr/StreamrBroadcaster.ts (hunk 1 above)
Comment (hatskier) [reply src/broadcasters/streamr/StreamrBroadcaster.ts]: Done! Even removed this private property. I've found simpler way to handle the timer running

File: src/NodeRunner.ts (hunk 2 above)
Comment (hatskier) [reply src/NodeRunner.ts:233]: I would leave the Promise.all as it would not kill the second promise if any of them fails

**Pull request merged**